library/src/main/java/com/xwray/groupie/Section.java

```diff
@@ -113,6 +113,24 @@ public void removeAll(@NonNull Collection<? extends Group> groups) {
         refreshEmptyState();
     }
 
+    /**
+     * Remove all existing body content.
+     */
+    public void clear() {
+        if (children.isEmpty()) {
+            return;
+        }
+
+        final List<Group> oldBodyGroups = new ArrayList<>(children);
+        super.removeAll(children);
```

Comment:
it seems to me you can leverage the current implementation of `Section.removeAll` and just do
```
        if (children.isEmpty()) {
            return;
        }
        removeAll(new ArrayList<>(children));
```
Which would have the same effect. If you agree, can you please modify the PR.

Reply:
Thanks for your review :) I'll change to it.